docs/word/word-add-ins-events.md

```diff
@@ -0,0 +1,214 @@
+
+The following code sample registers an event handler for the `onParagraphChanged` event in the document and defines the `handleChange` function that will run when the event occurs. It also defines the `deregisterEventHandler()` function that can subsequently be called to remove that event handler. Note that the `RequestContext` used to create the event handler is needed to remove it.
+
+```js
+let eventContext;
+
+async function registerEventHandler() {
+  await Word.run(async (context) => {
+    eventContext = context.document.onParagraphChanged.add(handleChange);
+
+    await context.sync();
+    console.log("Event handler successfully registered for onParagraphChanged event in the document.");
+  });
+}
+
+async function handleChange(event: Word.ParagraphChangedEventArgs) {
+  await Word.run(async (context) => {
+    await context.sync();
+    console.log(`${event.type} event was detected.`);
+  });
+}
+
+async function deregisterEventHandler() {
+  // The `RequestContext` used to create the event handler is needed to remove it.
+  // In this example, `eventContext` is being used to keep track of that context.
+  await Word.run(eventContext.context, async (context) => {
+    eventContext.remove();
+    await context.sync();
+    
+    eventContext = null;
+    console.log("Removed event handler that was tracking content changes in paragraphs.");
+  });
+}
+```
+
+## Use .track()
+
+Certain event types also require you to call `track()` on the object you're adding the event to.
+
+- Content control events
+  - `onDataChanged`
+  - `onDeleted`
+  - `onEntered`
+  - `onExited`
+  - `onSelectionChanged`
+- Comment events (preview)
+  - `onCommentAdded`
+  - `onCommentChanged`
+  - `onCommentDeleted`
+  - `onCommentDeselected`
+  - `onCommentSelected`
+
+The following code sample shows how to register an event handler on each content control. Because you're adding the event to the content controls, `track()` is called on each content control in the collection.
+
+```typescript
+let eventContexts = [];
+
+await Word.run(async (context) => {
+  const contentControls: Word.ContentControlCollection = context.document.contentControls;
+  contentControls.load("items");
+  await context.sync();
+
+  // Register the onDeleted event handler on each content control.
+  if (contentControls.items.length === 0) {
+    console.log("There aren't any content controls in this document so can't register event handlers.");
+  } else {
+    for (let i = 0; i < contentControls.items.length; i++) {
+      eventContexts[i] = contentControls.items[i].onDeleted.add(contentControlDeleted);
+
+      // Call track() on each content control.
+      contentControls.items[i].track();
+    }
+
+    await context.sync();
+
+    console.log("Added event handlers for when content controls are deleted.");
+  }
+});
+```
+
+The following code sample shows how to register comment event handlers on the document's body object and include a `body.track();` statement.
```

Comment:
Nit: Personal preference on excluding semicolons when talking about code in text.
```suggestion
The following code sample shows how to register comment event handlers on the document's body object and include a `body.track()` statement.
```